x-pack/test/functional/apps/saved_objects_management/import_saved_objects_between_versions_6.x_7.x.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* This test is importing saved objects from 6.8.x to 7.x.
+ * Please note a separate pr handled this for 7.x to 8.0.0 https://github.com/elastic/kibana/pull/91908
+ */
+
+import expect from '@kbn/expect';
+import path from 'path';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects(['common', 'settings', 'header', 'savedObjects']);
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+
+  describe('Export import saved objects between versions - 6.8.x -> 7.x', function () {
+    beforeEach(async function () {
+      await esArchiver.load('empty_kibana');
+      await esArchiver.load('logstash_functional');
+      await esArchiver.load('getting_started/shakespeare');
```

Comment:
Do we actually need to load 'logstash_functional' and 'getting_started/shakespeare' data?  I don't think importing the saved objects care if any data in any time range are used by the saved objects?

Reply:
We found out that we have to load the data otherwise Kibana will crash.

Reply:
Raised this bug - https://github.com/elastic/kibana/issues/97032